wildfly-standalone-centos8/README.md

```diff
@@ -48,16 +46,15 @@ Following is the Architecture:
 
 ![alt text](images/wildfly-arch.png)
 
-To learn more about the WildFly 18.0.0.Final, visit:
-https://docs.wildfly.org/18/
+To learn more visit the [Wildfly Documentation](https://docs.wildfly.org/18/) site.
 
 ## Licenses, Subscriptions and Costs 
```

Comment:
5.	Change accommodated – Heading changed to “Subscriptions and Costs” and paragraph under this section